### ISSUE TITLE
[DOC] The command description for Building on local in the README is incorrect.

### ISSUE THREAD

**Issue by MikuSugar** [alibaba/GraphScope#2613]
https://github.com/alibaba/GraphScope#building-on-local

The original text is
```
./gs install_deps dev

# With argument --cn to speed up the download if you are in China.
./gs install_deps dev --cn
```
The execution will result in an error.
![image](https://user-images.githubusercontent.com/27965111/232645451-92b1933b-3759-4094-9c59-3655e2d416e8.png)
Looking through other documents, you will know that the correct command is
```
./gs install-deps dev
./gs install-deps dev --cn
```

**Comment by MikuSugar** [comment]
The reference document is other documents. https://github.com/alibaba/GraphScope/blob/6fcca8881e419d8b4d5f14c66f724a740ed03b00/docs/development/dev_guide.md#install-deps-on-local

**Comment by siyuan0322** [comment]
Oh it should be `install-deps` it's a hypen `-`

**Comment by siyuan0322** [comment]
> The reference document is other documents. > https://github.com/alibaba/GraphScope/blob/6fcca8881e419d8b4d5f14c66f724a740ed03b00/docs/development/dev_guide.md#install-deps-on-local

This document is the correct usage.

**Comment by siyuan0322** [comment]
Has a fix for that #2614, thanks for reporting.
